bin/alluxio-start.sh

```diff
@@ -37,7 +37,8 @@ MOPT (Mount Option) is one of:
   SudoMount\tMount the configured RamFS using sudo.
            \tNotice: this will format the existing RamFS.
   NoMount  \tDo not mount the configured RamFS.
-           \tNotice: Use NoMount (Linux only) to use tmpFS to avoid sudo requirement.
+           \tNotice: to avoid sudo requirement but use tmpFS in Linux,
+             one can set ALLUXIO_RAM_FOLDER=/dev/shm and use NoMount.
```

Comment:
How is `ALLUXIO_RAM_FOLDER` expected to be used when you have remote workers? Will remote workers be able to use `/dev/shm`?